app/models/volunteer_application.rb

```diff
@@ -0,0 +1,21 @@
+class VolunteerApplication < ActiveRecord::Base
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  VALID_PHONE_REGEX = /[0-9]{10}/
```

Comment:
I'd consider pulling this out into it's own validator. It's not necessary per say, but I think it could be good practice and it will keep this model leaner. If you have a custom validator you should be able to do

```ruby
validates :cell, phone: true, presence: true
```

Check out http://guides.rubyonrails.org/active_record_validations.html#performing-custom-validations to see how rails suggests doing this. If you decide to create a custom validator, I'd suggest throwing it into `app/validators`